opencivicdata/python-opencivicdata
# Migrate from Travis to GH Actions

## Description

This PR migrates CI from Travis to GitHub Actions and adds a build step to package and upload to PyPI from tags.

### Testing instructions

- Confirm CI passes.
- Merge, then prep for new release and push a tag. Confirm the new dist is uploaded to PyPI.
